jenkins2/README.md

```diff
@@ -0,0 +1,60 @@
+This setup is based on the tutorial you can find here:
+
+http://engineering.riotgames.com/news/building-jenkins-inside-ephemeral-docker-container
+
+There several key changes however that won't match that blog. Namely:
+
+1. This uses Jenkins 2.7
```

Comment:
2.22 is used in docker images if i understand correctly. Can suggest suggest for end users latest LTS.